app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -28,7 +28,7 @@ There are some teacher training courses that come with a salary.
 
 This means you will not have to pay tuition fees and will receive a salary while you train to get [qualified teacher status (QTS)](/what-is-qts). This is what you need to teach in many primary, secondary and special schools in England. 
 
-The main salaried courses include School Direct (salaried), postgraduate teaching apprenticeships (PGTA) and teacher training delivered by Teach First.
+The main salaried courses include School Direct salaried, postgraduate teaching apprenticeships (PGTA) and teacher training delivered by Teach First.
 
 All salaried courses are broadly the same, but the course length and qualifications you work towards may vary.
 
```

Comment:
Do you need to remove the brackets on line 65 as well? (sorry, can't add a comment directly there!)